fiftyone/factory/repos/delegated_operation.py

```
@@ -218,25 +217,20 @@ def queue_operation(self, **kwargs: Any) -> DelegatedOperationDocument:
-            # If the dataset_id is provided, we set it in the request_params
-            # to ensure that the operation is executed on the correct dataset
-            context.request_params["dataset_id"] = str(op.dataset_id)
-            context.request_params["dataset_name"] = context.dataset.name
+
+        # For consistency, set the dataset_id using the
+        # ExecutionContext.dataset
+        # rather than calling load_dataset() on a potentially stale
+        # dataset_name in the request_params
+        try:
+            op.dataset_id = context.dataset._doc.id
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)